Mathlib/Topology/ContinuousFunction/FunctionalCalculus.lean

```diff
@@ -343,9 +349,21 @@ lemma cfc_id' : cfc (fun x : R ↦ x) a = a := cfc_id R a
 lemma cfc_map_spectrum : spectrum R (cfc f a) = f '' spectrum R a := by
   simp [cfc_apply f a, cfcHom_map_spectrum (p := p)]
 
+lemma cfc_const (r : R) (a : A) (ha : p a := by cfc_tac) :
+    cfc (fun _ ↦ r) a = algebraMap R A r := by
+  rw [cfc_apply (fun _ : R ↦ r) a, ← AlgHomClass.commutes (cfcHom ha (p := p)) r]
+  congr
+
 lemma cfc_predicate : p (cfc f a) :=
   cfc_apply f a ▸ cfcHom_predicate (A := A) ha _
 
```

Comment:
Can you remove the `p a` and continuity conditions on this one, now that we know that `p` is true on the junk value? (And the same for the non-unital case)